docs: polish quickstart (#303)
kurtosis-tech/kurtosis: docs/docs/quickstart.md

If you're ready to get going, this guide will take ~15 minutes and will walk you through building a basic Kurtosis package. The package that you build will start a Postgres server, seed it with data, put an API in front of it, and automate loading data into it.

#### Setup

> Before you proceed, make sure you have [Kurtosis installed][installing-kurtosis-guide] (or [upgraded to latest][upgrading-kurtosis-guide] if you already have it) and Docker is started and a Docker daemon is running. Other than these two items, you do not need any other knowledge.

Reviewer: ```suggestion
Before you proceed, make sure you have [Kurtosis installed][installing-kurtosis-guide] (or [upgraded to latest][upgrading-kurtosis-guide] if you already have it), Docker is started, and the Docker engine is running.
```
Think this can be trimmed

    )
```

> In `postgres_url`, replace that `"postgres"` with `POSTGRES_USER` in your `main.star` file to use the correct username we specified at the beginning of our file, and then rerun your dev loop:

Reviewer: ```suggestion
In the line declaring the `postgres_url` variable in your `main.star` file, replace the `"postgres"` string with `POSTGRES_USER` to use the correct username we specified at the beginning of our file. Then rerun your dev loop:
```
